test/unit/react-imgix.test.jsx

```diff
@@ -309,6 +310,56 @@ describe("When in <source> mode", () => {
     const onMountArg = onMountedSpy.lastCall.args[0];
     expect(onMountArg).toBeInstanceOf(HTMLSourceElement);
   });
+
+  describe("using the htmlAttributes prop", () => {
+    it("assigns an alt attribute given htmlAttributes.alt", async () => {
+      const htmlAttributes = {
+        alt: "Example alt attribute"
+      };
+
+      sut = mount(
+        <Source
+          src={"https://mysource.imgix.net/demo.png"}
+          sizes="100vw"
+          htmlAttributes={htmlAttributes}
+        />
+      );
+
+      expect(sut.props()["htmlAttributes"].alt).toEqual(htmlAttributes.alt);
+    });
+
+    it("passes any attributes via htmlAttributes to the rendered element", () => {
+      const htmlAttributes = {
+        "data-src": "https://mysource.imgix.net/demo.png"
+      };
+      sut = mount(
+        <Source
+          src={"https://mysource.imgix.net/demo.png"}
+          sizes="100vw"
+          htmlAttributes={htmlAttributes}
+        />
+      );
+
+      expect(sut.props()["htmlAttributes"]["data-src"]).toEqual(
+        htmlAttributes["data-src"]
+      );
+    });
+
+    it("attaches a ref via htmlAttributes", () => {
```

Comment:
We should add a test here for when `htmlAttributes.ref` is used, that onMounted is also called.